syntax = "proto3";

message PersonMessage {
    string first_name = 1;
    string last_name = 2;
    int32 id = 3;
    string company = 4;
}

service PersonService {
    rpc storePerson(PersonMessage) returns (PersonMessage) {}
}




